### DISCUSSION TITLE
Importing downloads and another error posted in the Health Status Tab

### DISCUSSION THREAD

**Discussion by JabbaDaHutt762** [RandomNinjaAtk/docker-lidarr-extended#115] [Q&A] [answered]
![health](https://user-images.githubusercontent.com/57692223/198885074-dc8e599c-d0c4-4fc7-97f4-492136733963.PNG)
![remote path](https://user-images.githubusercontent.com/57692223/198885075-c535e1da-f238-4106-a0c1-7b501d9a7dee.PNG)
![lidarr paths](https://user-images.githubusercontent.com/57692223/198885721-6ec891b7-4a39-4512-9d8c-c96de176e3ce.PNG)

Yellow error about the image im not sure if there's anything i need to do to resolve.

Red error I cant seem to get the completed downloads to import.  It shows in most cases that it was imported but when checking the library both in the file system and in the library on Lidarr the new files do not exist.  they are in the download folder though.  Im not having to use remote path mappings with any of the other arrs.  they just work.  what am i doing wrong?  something in the call out of the paths in the lidarr setup or this just a common issue with lidarr?

**Comment by RandomNinjaAtk** [comment, marked as answer]
Yellow error you can ignore. The import issues are likely caused by incorrect volume mappings.

Also if your volume mappings are correct, there is no need to use the remote path mapping feature.

**Reply by JabbaDaHutt762** [reply]
Thats what i would have thought too.  it just doesnt seem to grab.  ill keep playing with it.  the music library is huge though so every time i change the path i have to rebuild the library and it takes a very long time.

**Reply by RandomNinjaAtk** [reply]
All I can tell you is its because you have an incorrect configuration. 

Without having all the details of both containers its difficult to conclude where you have a miss configuration, but either way it is a user miss configuration issue.

My suggestion is read the errors carefully and then use CLI to verify the error to better understand the issue...
